sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotDecreaseMemberVisibility.CSharp9.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
```

Comment:
this is the existing test for classes where I replaced:
- `class` to `record`
- `set` to `init`

it's a bit overkill as records are just classes, but it shouldn't hurt.

Reply:
It's `Implement`, so it's useful like this